modules/services/repositories/src/test/java/au/com/shiftyjelly/pocketcasts/repositories/categories/CategoriesManagerTest.kt

```diff
@@ -23,6 +23,9 @@ internal class CategoriesManagerTest {
     @get:Rule
     val featureFlagRule = InMemoryFeatureFlagRule()
 
+    @get:Rule
+    val coroutineRule = MainCoroutineRule()
```

Comment:
I'm not sure I understand the need for this rule. This rule is generally necessary for `ViewModel`s because of the problematic `viewModelScope` property, which enforces `Dispatchers.Main`. 

In this case, however, the rule doesn't add value to the test. It would be simpler to just use:

```kotlin
private val scope = CoroutineScope(UnconfinedTestDispatcher())

@After
fun tearDown() {
  scope.cancel()
}
```